docs/quickstart-guide.md

```diff
@@ -0,0 +1,169 @@
+
+Open `app/views/admin/articles/_form.html.slim` and focus on the `main.content` section where the form fields are:
+
+```ruby
+main.content
+  = fae_input f, :title
+  = fae_input f, :slug
+  = fae_input f, :introduction
+  = fae_input f, :body
+  = fae_input f, :date
+
+  = fae_association f, :article_category
+
+  = fae_image_form f, :hero_image
+  = fae_file_form f, :pdf
+```
+
+
+**Automatically generate the `slug` from content in the `title`.**
+
+Fae's form helpers offer many options to help you customize. Let's add a class to the `title` field to trigger Fae's slugger feature:
+
+```ruby
+= fae_input f, :title, input_class: 'slugger'
+```
+
+_[Learn more about Fae's slugger](https://www.faecms.com/documentation/features-slugger)_
+
+**Add a markdown editor to `body`.**
+
+```ruby
+= fae_input f, :body, markdown: true
+```
+
+**Add a datepicker to `date`.**
+
+Fae also has a number of other form helpers that provide a variety of UI elements.
+
+```ruby
+fae_datepicker f, :date
+```
+
+_[See documentation on all form helpers](https://www.faecms.com/documentation/helpers-form_helpers)_
+
+## Form Validations
+
+Fae's forms will honor any standard validation rules defined in the model. Fae even has some [helpers for common validation scenerios](https://www.faecms.com/documentation/topics-models#validation).
+
+To add form validations to our articles, we just need to define them in `app/models/article.rb`
+
+```ruby
+validates :title, presence: true
+validated :slug, Fae.validation_helpers.slug
+```
+
+Fae uses [Judge](https://github.com/joecorcoran/judge) for client side validations. The caveat is Judge requires you to expose any attributes that have a uniqueness validation. You can do this in `config/initializers/jugde.rb`:
+
+```ruby
+Judge.configure do
+  expose Article, :slug
+end
+```
+
+## Navigation
+
+We should update our navigation so these three sections are easy to find. We'll update our `app/models/concerns/fae/navigation_concern.rb` like so:
+
+```ruby
+def structure
+  [
+    item('News', subitems: [
+      item('Articles', path: admin_articles_path),
+      item('Article Categories', path: admin_article_categories_path),
+    ]),
+    item('Pages', subitems: [
+      item('About Us', path: fae.edit_content_block_path('about_us'))
+    ])
+  ]
+end
+```
+
+_[Learn more about setting up your navigation](https://www.faecms.com/documentation/topics-navigation_setup)_
+
+## Scale and Customize
+
+Now that we are up and running with our CMS we can continue to scale it. When you run into a use case that Fae's can't handle, don't fret, Fae is built to be overridden.
+
+Use Rails helpers in place of Fae form helpers, override views, methods and whole classes. Fae even provides concerns to inject custom logic into it's base classes.
+
+We want Fae to get you off the ground quickly but be able to scale with your project and all the customized use cases you don't know you need yet.
```

Comment:
_bows_ We're still figuring that out, faecms.com pulls documentation from here, but I think we need to write some sort of converter so doc links work for both site and GH.